src/components/InteractiveSelection.js

```diff
@@ -125,16 +125,22 @@ class InteractiveSelection extends AbstractSelection {
   }
 
   stopDragSelection() {
+    const {onAreaUpdate} = this.props;
```

Comment:
The code across the project uses spaces for object literals. My bad that the project does not have a eslint config, but please, for the sake of consistency, use spaces with object literals/destructuring. Here's the eslint rule for the reference: https://eslint.org/docs/rules/object-curly-spacing

Reply:
No problem, I will update ASAP

Reply:
done